lib/core.fs

```diff
@@ -406,6 +406,25 @@ ps-drop,
 ret,
 end-code
 
+( -- addr )
+code here
+ps-dup,
+$C000 ]* A ld, A H ld,
```

Comment:
Also, perhaps `here` and `allot` are a bit too high level.

Couldn't we just expose `dp and dp!` words to set the dictionary pointer? then allot and here are easily defined in Forth.

Reply:
We can do that as well, but I noticed that for example gforth does not seem to expose the `dp` directly. Not sure if there was some reasoning behind that.

Reply:
well, I mean, exposing them from the kernel, but defining the higher level ones in the lib/ in Forth. But anyway it is probably more efficient in asm so it's fine.